lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1357,6 +1357,7 @@ enum DebuggerBroadcastBit {
   eBroadcastBitError = (1 << 2),
   eBroadcastSymbolChange = (1 << 3),
   eBroadcastBitProgressCategory = (1 << 4),
+  eBroadcastBitExternalProgressCategory = (1 << 5),
```

Comment:
You'll also need to add a `eBroadcastExternalBitProgress` unless you only ever want to listen for aggregated progress events (which is the difference between `eBroadcastBitProgressCategory` and `eBroadcastBitProgress`).